app/controllers/providers/means/capital_disregards/mandatory_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Providers
+  module Means
+    module CapitalDisregards
+      class MandatoryController < ProviderBaseController
+        prefix_step_with :capital_disregards
+
+        def show
+          @form = Providers::Means::CapitalDisregards::MandatoryForm.new(model: legal_aid_application)
+        end
+
+        def update
+          @form = Providers::Means::CapitalDisregards::MandatoryForm.new(form_params)
+          render :show unless save_continue_or_draft(@form)
+        end
+
+      private
+
+        def form_params
+          merge_with_model(legal_aid_application) do
```

Comment:
i do not believe you need this `merge_with_model` block - i removed it from discretionary - as we explicitly extract legal_aid_application in the intializer of the form...but am wondering if the update depends on it now 🤔